<div class="intro-box">
  <div class="row">
    <div class="columns small-12 medium-10 medium-offset-1">
      <div class="intro-content">
        The 3D-Beacons Network provides unified programmatic access to experimentally determined and predicted structure models.
      </div>
    </div>
  </div>
</div>

<main class="main-content" id="main-content">
</main>

<div class="row home-row">
  <div class="small-12 medium-10 large-8 columns" style="display:flex; margin-left:30px; margin-bottom:40px; min-height:63px; border-left:3px solid #085F5C">
    <h2 class="heading1">Background</h2>
  </div>

  <div class="small-10 columns" style="display:flex; margin-left:10px; float:left">
    <p class="text1" style="margin-top: 4px;">3D-Beacons is an open collaboration between providers of macromolecular structure models. This collaboration aims to provide model coordinates and meta-information from all the contributing data resources in a standardised data
    format, on a unified platform.</p>
  </div>

  <div class="small-10 columns" style="display:flex; margin-left:10px; float:left">
    <p class="case-study-p">All of the data provided is freely available for both academic and commercial use under Creative Commons Attribution 4.0 (<a href="https://creativecommons.org/licenses/by/4.0/" target="_blank">CC-BY 4.0</a>) license terms. If you use this resource, please cite the following papers:
      <a href="https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giac118/6854872" target="_blank">
        M. Varadi, S. Nair, I. Sillitoe <i>et al</i>. 3D-Beacons: decreasing the gap between protein sequences and structures through a federated network of protein structure data resources. <i>GigaScience</i> (2022).
      </a>
    </p>
  </div>

  <div class="small-10 columns" style="display:flex; margin-left:10px; float:left">
    <p class="case-study-p">
      The 3D Beacons API retrieves and combines data from member data providers, and returns the data in JSON format according to the 3D-Beacons API specification.
    </p>
  </div>
</div>

<div class="row home-row">
  <div class="small-12 medium-10 large-8 columns" style="display:flex; margin-left:30px; margin-bottom:40px; min-height:63px; border-left:3px solid #085F5C">
    <h2 class="heading1">How to use it?</h2>
  </div>
  <div class="columns small-12 medium-10" style="display:flex; margin-left:10px;">
    <p class="text1" style="margin-top: 4px;">The main purpose of 3D-Beacons is to provide programmatic access to experimentally determined and theoretical
      protein structures.</p>
  </div>
  <div class="columns small-12 medium-10" style="display:flex; margin-left:10px;">
    <p class="case-study-p">The <a href="https://www.ebi.ac.uk/pdbe/pdbe-kb/3dbeacons/api/" target="_blank">3D Beacons API</a>
      retrieves and combines data from member data providers, and returns the data in JSON format according to the
      <a href="https://github.com/3D-Beacons/3d-beacons-specifications/blob/main/oas3.yaml" target="_blank">3D-Beacons API specification</a>.
    </p>
  </div>
  <div class="columns small-12 medium-10" style="display:flex; margin-left:15px; float:none;">

    <a href="https://www.ebi.ac.uk/pdbe/pdbe-kb/3dbeacons/api/" target="_blank">
    <button class="vf-button vf-button--secondary">
      <span class="vf-button__text">See the 3D-Beacons HUB API</span>
    </button>
    </a>

    <!-- <a href="https://www.ebi.ac.uk/pdbe/pdbe-kb/3dbeacons/api/" target="_blank">
      <button class="api-link vf-search__button | vf-button vf-button--secondary" style="margin-top: 4px; margin-bottom:25px;">
        <span class="hide-for-small-only vf-button__text button-text">See the 3D-Beacons HUB API</span>
        <svg class="show-for-small-only vf-icon vf-icon--search-btn | vf-button__icon" aria-hidden="true"
          xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink"
          xmlns:svgjs="http://svgjs.com/svgjs" viewBox="0 0 140 140" width="16" height="16">
          <g transform="matrix(5.833333333333333,0,0,5.833333333333333,0,0)">
            <path
              d="M23.414,20.591l-4.645-4.645a10.256,10.256,0,1,0-2.828,2.829l4.645,4.644a2.025,2.025,0,0,0,2.828,0A2,2,0,0,0,23.414,20.591ZM10.25,3.005A7.25,7.25,0,1,1,3,10.255,7.258,7.258,0,0,1,10.25,3.005Z"
              fill="#397F7D" stroke="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="0"></path>
          </g>
        </svg>
      </button>
    </a> -->
  </div>
</div>


<div class="row home-row">
  <div class="small-12 medium-10 large-8 columns" style="display:flex; margin-left:30px; margin-bottom:40px; min-height:63px; border-left:3px solid #085F5C">
    <h2 class="heading1">How does it work?</h2>
  </div>
  
  <div class="columns small-12 medium-10" style="display:flex; margin-left:10px;">
    <p class="text1" style="margin-top: 4px;">
      The 3D-Beacons Network consists of a Registry, a Hub and data providers who host Clients. The
      <a href="https://github.com/3D-Beacons/3d-beacons-hub-api" target="_blank">3D-Beacons
        Hub</a> uses the <a href="https://github.com/3D-Beacons/3d-beacons-registry" target="_blank">
        3D-Beacons Registry</a> to look up which
      <a href="https://github.com/3D-Beacons/3d-beacons-client" target="_blank">3D-Beacon Client</a>
      supports what API endpoints. The Beacons provide data according to the
      <a href="https://github.com/3D-Beacons/3d-beacons-specifications/blob/main/oas3.yaml" target="_blank">
        3D-Beacons data specifications</a>. The Hub collates the data from
      the Beacons and exposes it via Hub API endpoints.
    </p>
  </div>
  <div class="columns small-12 medium-10" style="display:flex; margin-left:10px; float:none;">
    <p class="small-12 medium-10 vf-section-header__heading--is-link">
      <a class="see-link" routerLink="/docs" fragment="infra">
      <span>See the Documentation page for more details</span>
        <svg aria-hidden="true" class="vf-section-header__icon | vf-icon vf-icon-arrow--inline-end" width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
          <path d="M0 12c0 6.627 5.373 12 12 12s12-5.373 12-12S18.627 0 12 0C5.376.008.008 5.376 0 12zm13.707-5.209l4.5 4.5a1 1 0 010 1.414l-4.5 4.5a1 1 0 01-1.414-1.414l2.366-2.367a.25.25 0 00-.177-.424H6a1 1 0 010-2h8.482a.25.25 0 00.177-.427l-2.366-2.368a1 1 0 011.414-1.414z" fill="" fill-rule="nonzero"></path>
        </svg>
      </a>
    </p>
  </div>
  <div class="columns small-12 medium-10" style="display:flex; margin-left:10px; float:none;">
    <img src="./assets/img/overview.png" alt="overview of the 3d-beacons infrastructure">
  </div>  
</div>

<section>
  <div class="row info-section first-section">
    <div class="columns small-12 medium-10">
      <h3>Case studies of integrating 3D-Beacons</h3>
      <div class="row">
        <div class="columns home-text case-study-div">
          <a href="https://www.jalview.org/" target="_blank">Jalview</a>
        </div>
        <div class="columns medium-6 info-content">
          <p class="case-study-p">
            3D Beacons allows users of the <a href="https://www.jalview.org/" target="_blank">Jalview workbench for multiple
              sequence alignment and analysis</a> to discover 3D models for alignments of proteins from Uniprot and place them
              in context of genetic variation from Ensembl. Its capabilities also include visualisation of local model quality
              scores (including method specific quantities such as pLDDT) and sequence alignment based superimposition of
              structures with Jmol, UCSF Chimera/X and Pymol.
              <br><br>
              For more information see <a
              href="https://www.jalview.org/help/html/features/structurechooser.html#3dbeaconssearch" target="_blank">Jalview's
              3D Beacons search documentation</a>. The Jalview application is available for a variety of platforms and can also
              be run <a href="https://www.jalview.org/jalview-js" target="_blank">in the browser</a>.
          </p>
        </div>

        <div class="columns medium-6 info-content" style="text-align: center;">
          <a href="https://www.jalview.org/" target="_blank">
            <img class="ex-img" src="./assets/img/jalview-example.png">
          </a>
        </div>

      </div>

      <div class="row info-section-row-2">
        <div class="columns home-text case-study-div">
          <a href="https://pdbe-kb.org" target="_blank">PDBe-KB</a>
        </div>
        
        <div class="columns medium-push-6 medium-6 info-content">
          <p class="case-study-p">
            The <a href="https://pdbe-kb.org" target="_blank">Protein Data Bank in Europe - Knowledge Base (PDBe-KB)</a>
            relies on the 3D-Beacons Hub API to display all the experimentally determined and theoretical structures for
            proteins of interest (<a href="https://www.ebi.ac.uk/pdbe/pdbe-kb/proteins/P38398/structures" target="_blank">example</a>).
            The so-called "aggregated views of proteins" pages of PDBe-KB collate all the available structural data from the
            PDB and place these structures in the context of their functional, biophysical and biochemical annotations
            provided by partner data resources. The integration of the 3D-Beacons Hub API allows us to display these
            annotations concerning theoretical models and the experimentally determined structures.
          </p>
        </div>

        <div class="columns medium-pull-6 medium-6" style="text-align: center;">
          <a href="https://pdbe-kb.org" target="_blank">
            <img src="./assets/img/pdbe-kb-example-alt.png" alt="example of how the PDBe-KB integrated 3D-Beacons">
          </a>
        </div>
      </div>

      <div class="row">
        <div class="columns home-text case-study-div">
          <a href="https://swissmodel.expasy.org/repository" target="_blank">SWISS-MODEL Repository</a>
        </div>
        <div class="columns medium-6 info-content">
          <p class="case-study-p">
            The <a href="https://swissmodel.expasy.org/repository" target="_blank">SWISS-MODEL Repository (SMR)</a> takes
            advantage of the 3D-Beacons API to fetch models from the AlphaFoldDB and ModelArchive for display in addition to
            homology models from SWISS-MODEL and experimental structures from the PDB (<a
            href="https://swissmodel.expasy.org/repository/uniprot/P53742?model=ma-bak-cepc-0746" target="_blank">example</a>).
            Within SMR, the models are displayed with a consistent colouring for model confidence to compare homology with
            AlphaFold models and structures can be further assessed with <a href="https://swissmodel.expasy.org/assess/help"
                                                                            target="_blank">other metrics</a>. Furthermore,
            SMR includes sequence based annotations which can be visualised on the structures and enables the <a
            href="https://swissmodel.expasy.org/comparison/help" target="_blank">comparison of multiple structures</a>.
          </p>
        </div>

        <div class="columns medium-6 info-content" style="text-align: center;">
          <a href="https://swissmodel.expasy.org/repository/uniprot/P53742?model=ma-bak-cepc-0746" target="_blank">
            <img src="./assets/img/swiss-model-example.png"
                 alt="example of how the SWISS-MODEL Repository integrated 3D-Beacons">
          </a>
        </div>

      </div>

    </div>
  </div>
</section>

<app-providers></app-providers>

